string task
float64 priority
int8 x
int8 y